source/Playnite.DesktopApp/ViewModels/GameEditViewModel.cs

```diff
@@ -174,6 +174,8 @@ public bool IsBackgroundTooLage
 
         #endregion Database fields
 
+        public bool IsHdrSupported => HdrUtilities.IsHdrSupported();

```

Comment:
I don't think this is needed. Especially since people sometimes transfer libraries between PCs and they wouldn't be able to toggle this option on PCs that don't support and then use it on others that do. The checkbox should be always enabled.

Reply:
Hmm. This is an interesting point and I see where you are coming from. The counterpoint to this (and why I implemented it this way) is that it may make users believe that they have access to features that their monitors don't actually support. I want to think about this a bit more.

Reply:
I'm pretty a user would know if they have HDR support available on their system prior to actually using Playnite. It would be very weird use case if Playnite would actually introduce them to HDR being available to them.

Reply:
I think usable solution would be adding some indicator next to this option saying that HDR support was detected or not detected on this system.

Reply:
Ok so I have tweaked this so that the `IsHdrSupported` check still exists, but its only purpose here is to enable me to show additional info in the tooltip:
![image](https://user-images.githubusercontent.com/6503344/218679221-3e967e76-a660-4540-8e50-5eb7c050eb56.png)

Likewise, this is what it would look like if my primary monitor does support HDR:
<img width="583" alt="image" src="https://user-images.githubusercontent.com/6503344/218679362-285116d6-f86d-4be2-816f-292c1db1ab11.png">